libmscore/excerpt.h

```diff
@@ -29,8 +29,6 @@ class XmlReader;
 //   @@ Excerpt
 //---------------------------------------------------------
 
-#include <QMultiMap>
```

Comment:
Please tell me, for understanding, why remove the includes of types that are used in the file? This does not affect the build speed. What other reasons could there be?

Reply:
It does influence compile speed, as all.h gets precompiled

Reply:
Yes, precompiled header all.h it does influence compile speed,   
but includes of types that are used in the file do not interfere with this.  
They are not inserted by the preprocessor because they are actually already inserted by precompiled header (the insert is protected by a guard).

Reply:
In that case the include here is just superfluos

Reply:
We don't directly inlcude any Qt or system headers anywhere in the MuseScore code

Reply:
OK, I changed my mind ;-)

Reply:
But moved it up a couple lines

Reply:
Darn, `#include <QMap>` and `#include <QMultiMap>` are both doing the very same thing:   `#include <qmap.h>`

Reply:
> OK, I changed my mind ;-) (or you changed my mind?)
> But moved it up a couple lines

Thank you, this is very cool :)